src/collections/Menu/index.d.ts

```diff
@@ -135,7 +135,7 @@ interface MenuItemProps extends ReactMouseEvents<HTMLElement> {
   color?: SemanticCOLORS;
 
   /** Shorthand for primary content. */
-  content?: string;
+  content?: any;
```

Comment:
This is actually `PropTypes.node` which is:

>Anything that can be rendered: numbers, strings, elements or an array (or fragment) containing these types.

https://facebook.github.io/react/docs/typechecking-with-proptypes.html